src/main/paradox/kubernetes-deployment.md

```diff
@@ -7,9 +7,47 @@ Once you've installed the deployment tool, `reactive-cli`, you can use the `rp`
 * [reactive-cli](https://github.com/lightbend/reactive-cli) installed
 * Applications built and published to a registry (production) or the local docker engine ([Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/))
 * `kubectl` installed and configured to point to your registry
+*  Kubernetes cluster with correctly configured Role-Based Access Control ([RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)). 
 
 > Using Minikube? Make sure you run `eval $(minikube docker-env)` before following the steps below. When you need your old environment, you can get it back with `eval $(minikube docker-env -u)`.
 
+### RBAC
+
+Many production clusters and recent versions of Minikube come with authorization feature called [Role-Based Access Control](https://kubernetes.io/docs/admin/authorization/rbac/). Basic idea is to give users certain permissions when working with the cluster. Sets of these permissions are called roles. Users are given roles using role bindings. Note that even a process running inside a pod belong to a special type of user called [Service Account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/). 
+
+An application built using Lightbend Orchestration most likely needs a permission to list pods running in its namespace. This is neccessary for service discovery to work correctly. Here's an example Role and a RoleBinding for a service account in the default namespace:
+
+```
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pod-reader
+rules:
+- apiGroups: [""] # "" indicates the core API group
+  resources: ["pods"]
+  verbs: ["get", "watch", "list"]
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: read-pods
+subjects:
+# Note the `name` line below. The first default refers to the namespace. The second refers to the service account name.
+# For instance, `name: system:serviceaccount:myns:default` would refer to the default service account in namespace `myns`
+- kind: User
```

Comment:
this keys on the authorization module, which I don't think is quite right.  expecting to see a ServiceAccount kind, which is then referenced in the deployment line.

see here as an example:  https://github.com/lightbend/k8s-explore/blob/master/quick/kube-state-metrics.yaml#L58

Reply:
Just noting that this is exactly copied from https://developer.lightbend.com/docs/akka-management/current/discovery.html#role-based-access-control

If it needs to be changed, we don't want the two sets of documentation to diverge. It would probably be better to have a single source for this information, and then link to it from relevant places.